sdk/keyvault/Azure.Security.KeyVault.Administration/tests/samples/AccessControlSampleSnippets.cs

```diff
@@ -66,13 +66,18 @@ public void GetRoleDefinitions()
             #endregion
         }
 
-        [Test]
+        [RecordedTest]
         [SyncOnly]
-        public async Task CreateRoleAssignment()
+        public void CreateRoleAssignment()
         {
             client = Client;
-            List<RoleDefinition> definitions = await Client.GetRoleDefinitionsAsync(RoleAssignmentScope.Global).ToEnumerableAsync().ConfigureAwait(false);
-            _roleDefinitionId = definitions.FirstOrDefault(d => d.RoleName == RoleName).Id;
+            Pageable<KeyVaultRoleDefinition> definitions = client.GetRoleDefinitions(KeyVaultRoleScope.Global);
+            List<KeyVaultRoleDefinition> allDefinitions = new List<KeyVaultRoleDefinition>();
```

Comment:
What's the point of putting these in a collection? LINQ should work over the pageable.